Makefile

```diff
@@ -26,7 +26,7 @@ dev.up:
 
 dev.provision:
 	docker exec -t edx.devstack.mysql /bin/bash -c 'mysql -uroot <<< "create database if not exists blockstore_db;"'
-	docker exec -t edx.devstack.blockstore /bin/bash -c 'source ~/.bashrc && make requirements && make migrate'
+	docker exec -t edx.devstack.blockstore /bin/bash -c 'source ~/.bashrc && source /blockstore/venv/bin/activate && make requirements && make migrate'
```

Comment:
The path needs to be adjustable via `VIRTUAL_ENV` so we shouldn't hard code it here. It should be possible to set it by passing it in `blockstore_environment` variable and then use ${VENV_BIN} before `/pip` and `/python`.

Reply:
The virtual env paths are different for production and devstack. From what I understand, in production, everything should be within the `/edx/app` folder, but for the devstack we put the venv directory in `/blockstore`. Would having a similar directory structure (i.e. `/edx/app/blockstore/venvs/blockstore`) in the devstack make sense?

Reply:
Yup, that is why we just need to pass in the right `VIRTUAL_ENV` value and the rest should work. In devstack the `VIRTUAL_ENV` value is passed in from the [Dockerfile](https://github.com/open-craft/blockstore/blob/master/Dockerfile-dev#L5). In production, we can set it to wherever the venv is from Ansible. Does the make file in production not see the environment variables specified in `blockstore_environment`?

Reply:
Makes sense. I missed to notice the `?` in front of `VIRTUAL_ENV` and assumed it was being hard-coded anyway. I've reverted the change and have added it in `blockstore_environment`.